app/views/organizations/staff/default_pet_tasks/index.html.erb

```diff
@@ -4,67 +4,44 @@
-                <th scope="col">
-                  <%= t('.species') %>
-                </th>
-                <th scope="col">
-                  <%= t('.due_after') %>
-                </th>
-                <th scope="col">
-                  <%= t('.recurring') %>
-                </th>
-                <th scope="col"></th>
-              </tr>
-            </thead>
-            <tbody>
-              <% @default_pet_tasks.each do |task| %>
-                <tr>
-                  <td>
-                    <%= task.name %>
-                  </td>
-                  <td>
-                    <%= task.description %>
-                  </td>
-                  <td>
-                    <%= task.human_enum_name(:species) %>
-                  </td>
-                  <td>
-                    <%= task.due_in_days ? pluralize(task.due_in_days, "day") : "" %>
-                  </td>
-                  <td>
-                    <%= task.recurring? ? "Yes" : "No" %>
-                  </td>
-                  <% if current_user.staff_account %>
-                    <td>
-                      <div class="text-right">
-                        <div class="d-flex align-items-center justify-content-end">
-                          <%= link_to 'Edit', edit_staff_default_pet_task_path(task), class: 'btn btn-warning m-2' %>
-                          <%= link_to 'Delete', staff_default_pet_task_path(task), class: 'btn btn-danger m-2', data: { turbo_method: "delete", turbo_confirm: t('.are_you_sure_delete') } %>
-                        </div>
-                      </div>
-                    </td>
-                  <% end %>
-                </tr>
-              <% end %>
-            </tbody>
-          </table>
+  <!-- Ransack Search Form -->
+   <section class="pb-5">
+      <div class="container">
+        <div class="row p-4 border rounded-5 mb-5 bg-white">
+          <div class="col-xl-12">
+            <%= search_form_for @q, url: staff_default_pet_tasks_path do |f| %>
+              <div class="row">
+                <div class="form-group mb-3 col-md-4">
+                  <%= f.label :name_cont, t('.name') %>
+                  <%= f.text_field :name_cont, class: "form-control", placeholder: "Enter Pet Name" %>
+                </div>
+                <div class="form-group mb-3 col-md-4">
+                  <%= f.label :species_eq, t('.species') %>
+                  <%= f.select :species_eq, Pet.species.keys.map { |type| [type.titleize, type] }, { include_blank: 'Select Species' }, class: "form-select" %>
+                </div>
+                <div class="form-group mb-3 col-md-4">
+                  <%= f.label :due_in_days, t('.due_after') %>
+                  <%= number_field_tag :due_in_days, params[:due_in_days], class: "form-control", placeholder: "Due After (Days)" %>
+                </div>
+                <div class="form-group mb-3 col-md-4">
+                  <%= f.label :recurring, t('.recurring') %>
```

Comment:
I think this should also be `recurring_eq` given we are looking for an equality match.